src/mongocxx/private/libbson.hh

```diff
@@ -65,7 +65,7 @@ class MONGOCXX_TEST_API scoped_bson_t {
     //
     // The internal bson_t is initialized if the optional is populated.
     //
-    scoped_bson_t(bsoncxx::stdx::optional<bsoncxx::document::view_or_value> doc);
+    explicit scoped_bson_t(bsoncxx::stdx::optional<bsoncxx::document::view_or_value> doc);
```

Comment:
This seems to do more than you mentioned in the pr and definitely changes the way scoped_bson_t's will be constructed from arguments.  Not saying if it's right or wrong, but probably wrong for this pr